javaparser-core/src/main/javacc/java_1_8.jj

```diff
@@ -256,13 +261,39 @@ SKIP :
 TOKEN_MGR_DECLS :
 {
     private List<Token> tokens = new ArrayList<Token>();
+    private CommentsCollection commentsCollection = new CommentsCollection();
 
     List<Token> getTokens() {
         return tokens;
     }
+    
+    CommentsCollection getCommentsCollection() {
+        return commentsCollection;
+    }
 
     private void CommonTokenAction(Token token) {
         tokens.add(token);
+
+        while (token.specialToken != null) {
```

Comment:
Everything seems well indented to me, can you point out the problem?
It's newly written, by the way.